src/lib/snack-bar/snack-bar-ref.ts

```diff
@@ -0,0 +1,41 @@
+import {OverlayRef} from '@angular2-material/core';
+import {Observable} from 'rxjs/Observable';
+import {Subject} from 'rxjs/Subject';
+
+// TODO(josephperrott): Implement onAction observable.
+
+
+/**
+ * Reference to a snack bar dispatched from the snack bar service.
+ */
+export class MdSnackBarRef<T> {
+  /** The instance of the component making up the content of the snack bar. */
+  readonly instance: T|any;
```

Comment:
Why is `any` in here? It effectively makes the type just `any`. For dialog, this is just `null` when there's no component instance.